api/src/stampy_chat/chat.py

```diff
@@ -18,6 +19,12 @@
 from stampy_chat.followups import StampyChain
 from stampy_chat.citations import make_example_selector
 
+from langsmith import Client
+if os.environ.get("LANGCHAIN_API_KEY"):
+    os.environ['LANGCHAIN_TRACING_V2'] = "true"
```

Comment:
can these be explicitly passed into the `Client()` constructor, rather than playing about with env variable?

Reply:
modified it in new [commit](https://github.com/StampyAI/stampy-chat/pull/132/commits/d94abb7d9908728577beb7377a1749368d2860e1)

Now it raises an exception cause the setup of the client must be done with a valid key. So LANGCHAIN_TRACING_V2 sould not be "true" when there is no LANGCHAIN_API_KEY cause then the invokation of the chain does not work